### DISCUSSION TITLE
E (335799) SERVER_HELP: send_file: Failed to read file: /sdcard/html/value

### DISCUSSION THREAD

**Discussion by qingfeng-zhang** [jomjol/AI-on-the-edge-device#3229] [Q&A]
may i ask why there have such error?

**Comment by Slider0007** [comment]
If you request http://IP-Address/value the system is trying to load this resource from SD card html folder in which the requested file is not available -> error

There is no negative effect to the process itself...

**Comment by qingfeng-zhang** [comment]
so where is wrong? SD don't have the file , how to generate it ?

**Reply by Slider0007** [reply]
Nothing is wrong. You don't need that file.

**Comment by qingfeng-zhang** [comment]
ok, how to remore such errors?

**Comment by Slider0007** [comment]
Do not manually request resources which are not available or just ignore the error
